projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -167,6 +168,8 @@ if (environment.requestedDeliveryDate) {
     PaymentMethodsModule,
     NotificationPreferenceModule,
     MyInterestsModule,
+    MyAccountV2NavigationModule,
+    MyAccountV2NavigationModule,
```

Comment:
[NITPICK]
typo: Unnecessary 2x module import

(however it's not a public API so not a big deal)

Reply:
done